test/tests/test_bash_functions.py

```diff
@@ -1,165 +1,249 @@
-
-@pytest.mark.parametrize('test_args', [
-    '-e DNSSEC="true" -e CUSTOM_CACHE_SIZE="0"',
-])
+    slow(
+        lambda: re.search(CONFIG_LINE, docker.run(_cat(DNSMASQ_CONFIG_LOC)).stdout)
+        is not None
+    )
+
+
+@pytest.mark.parametrize(
+    "test_args",
+    [
+        '-e DNSSEC="true" -e CUSTOM_CACHE_SIZE="0"',
+    ],
+)
 def test_dnssec_enabled_with_custom_cache_size(docker, slow, test_args):
-    CONFIG_LINE = r'cache-size\s*=\s*10000'
+    CONFIG_LINE = r"cache-size\s*=\s*10000"
 
     docker.run(CMD_SETUP_FTL_CACHESIZE)
-    slow(lambda: re.search(CONFIG_LINE, docker.run(_cat(DNSMASQ_CONFIG_LOC)).stdout) != None)
-
-
-@pytest.mark.parametrize('args_env, expected_stdout, expected_config_line', [
-    ('', 'binding to default interface: eth0', 'PIHOLE_INTERFACE=eth0'),
-    ('-e INTERFACE="br0"', 'binding to custom interface: br0', 'PIHOLE_INTERFACE=br0'),
-])
-def test_dns_interface_override_defaults(docker, slow, args_env, expected_stdout, expected_config_line):
-    ''' When INTERFACE environment var is passed in, overwrite dnsmasq interface '''
+    slow(
+        lambda: re.search(CONFIG_LINE, docker.run(_cat(DNSMASQ_CONFIG_LOC)).stdout)
+        is not None
+    )
+
+
+@pytest.mark.parametrize(
+    "args_env, expected_stdout, expected_config_line",
+    [
+        ("", "binding to default interface: eth0", "PIHOLE_INTERFACE=eth0"),
+        (
+            '-e INTERFACE="br0"',
+            "binding to custom interface: br0",
+            "PIHOLE_INTERFACE=br0",
+        ),
+    ],
+)
+def test_dns_interface_override_defaults(
+    docker, slow, args_env, expected_stdout, expected_config_line
+):
+    """When INTERFACE environment var is passed in, overwrite dnsmasq interface"""
     function = docker.run(CMD_SETUP_FTL_INTERFACE)
     assert expected_stdout in function.stdout
-    slow(lambda: expected_config_line + '\n' == docker.run('grep "^PIHOLE_INTERFACE" {}'.format(SETUPVARS_LOC)).stdout)
+    slow(
+        lambda: expected_config_line + "\n"
+        == docker.run('grep "^PIHOLE_INTERFACE" {}'.format(SETUPVARS_LOC)).stdout
+    )
 
 
 expected_debian_lines = [
     '"VIRTUAL_HOST" => "127.0.0.1"',
-    '"PHP_ERROR_LOG" => "/var/log/lighttpd/error-pihole.log"'
+    '"PHP_ERROR_LOG" => "/var/log/lighttpd/error-pihole.log"',
 ]
 
 
-@pytest.mark.parametrize('expected_lines,repeat_function', [
-    (expected_debian_lines, 1),
-    (expected_debian_lines, 2)
-])
+@pytest.mark.parametrize(
+    "expected_lines,repeat_function",
+    [(expected_debian_lines, 1), (expected_debian_lines, 2)],
+)
 def test_debian_setup_php_env(docker, expected_lines, repeat_function):
-    ''' confirm all expected output is there and nothing else '''
+    """confirm all expected output is there and nothing else"""
     for _ in range(repeat_function):
-        docker.run('. /usr/local/bin/bash_functions.sh ; eval `grep setup_php_env /usr/local/bin/_startup.sh`').stdout
```

Comment:
I did remove this `.stdout` since its return value isn't being used. I think SonarCloud might have this one flagged as a code smell.